src/liballoc/boxed.rs

```diff
@@ -194,7 +194,9 @@ impl<T: ?Sized> Box<T> {
     }
 
     /// Consumes and leaks the `Box`, returning a mutable reference,
-    /// `&'a mut T`. Here, the lifetime `'a` may be chosen to be `'static`.
+    /// `&'a mut T`. Note that the type `T` must outlive the chosen lifetime
+    /// `'a`. If the type has only static references, or none at all, then this
+    /// may be chosen to be `'static`.
```

Comment:
isn't this a must, not a may?

Reply:
No, it's variant in the lifetime, so it can be shorter. Only `T: 'static` can leak as `&'static mut T`, but it can also leak as any other lifetime.